releases/2.0ga.md

```diff
@@ -0,0 +1,155 @@
+    - Reduce the scheduling overhead of the balance-region scheduler
+    - Optimize the execution efficiency of the the hot-region scheduler
+- Operations interface and configuration
+    - Support TLS
+    - Support prioritizing the PD leaders
+    - Support configuring the scheduling policies based on labels
+    - Support configuring stores with a specific label not to schedule the Raft leader
+    - Support splitting Region manually to handle the hotspot in a single Region
+    - Support scattering a specified Region to manually adjust Region distribution in some cases
+    - Add check rules for configuration parameters and improve validity check of the configuration items
+- Debugging interface
+    - Add the `Drop Region` debugging interface
+    - Add the interfaces to enumerate the health status of each PD
+- Statistics
+    - Add statistics about abnormal Regions
+    - Add statistics about Region isolation level
+    - Add scheduling related metrics
+- Performance 
+    - Keep the PD leader and the etcd leader together in the same node to improve write performance
+    - Optimize the performance of Region heartbeat
+
+## TiKV
+
+- Features
+    - Protect critical configuration from incorrect modification
+    - Support `Region Merge` [experimental]
+    - Add the `Raw DeleteRange` API
+    - Add the `GetMetric` API
+    - Add `Raw Batch Put`, `Raw Batch Get`, `Raw Batch Delete` and `Raw Batch Scan`
+    - Add Column Family options for the RawKV API and support executing operation on a specific Column Family
+    - Support Streaming and Streaming Aggregation in Coprocessor
+    - Support configuring the request timeout of Coprocessor
+    - Carry timestamps with Region heartbeats
+    - Support modifying some RocksDB parameters online, such as `block-cache-size`
+    - Support configuring the behavior of Coprocessor when it encounters some warnings or errors
+    - Support starting in the importing data mode to reduce write amplification during the data importing process
+    - Support manually splitting Region in halves
+    - Improve the data recovery tool `tikv-ctl`
+    - Return more statistics in Coprocessor to guide the behavior of TiDB
+    - Support the `ImportSST` API to import SST files [experimental]
+    - Add the TiKV Importer binary to integrate with TiDB Lightning to import data quickly [experimental]
+- Performance
+    - Optimize read performance using `ReadPool` and increase the `raw_get/get/batch_get` by 30%
+    - Improve metrics performance
+    - Inform PD immediately once the Raft snapshot process is completed to speed up balancing
+    - Solve performance jitter caused by RocksDB flushing
+    - Optimize the space reclaiming mechanism after deleting data
+    - Speed up garbage cleaning while starting the server
+    - Reduce the I/O overhead during replica migration using `DeleteFilesInRanges`
+- Stability
+    - Fix the issue that gRPC call does not returned when the PD leader switches
```

Comment:
Fix the issue that gRPC call does not **get** returned when the PD leader switches?